### DISCUSSION TITLE
EXR support

### DISCUSSION THREAD

**Discussion by GlacierFox** [google/model-viewer#4557] [Q&A]
I've been trying to get my collection of .exr files working with <model-viewer>  but I can't seem to get them to load in like my other .hdr files. According [this](https://github.com/google/model-viewer/issues/200) page their implementation has been completed. Is this correct?

**Comment by elalish** [comment]
Sorry, not completed - it was closed because we decided not to. EXR files are dramatically more complex and hard to support in a small web library. I'd recommend exporting them to .hdr.
